### PR TITLE
finishing title-requirements, add translation for new skills, and some small stuff

### PR DESCRIPTION
**Pull request by stardustichi**

### CODE REVIEW

File: src/data/title-requirements.csv [hunk 1]
```diff
@@ -1,138 +1,138 @@
...
+"作戦「逃げ」で6勝し、\n基礎能力[スピード]が600以上になる","Win a race 6 times as a runner\nReach 600 or higher speed"
+"作戦「差し」または「追込」でGⅠを勝利し、\n基礎能力[パワー]と[根性]がそれぞれ500以上になる","Win a GI race once as a betweener or chaser\nReach 500 or higher power and guts"
+"中央10か所のレース場で、それぞれ重賞を勝利する","Win a big prize each in 10 central race grounds"
+"日本ダービーを勝利する","Win the Japan Derby"
+"オークスを勝利する","Win the Oaks"
+"皐月賞、日本ダービー、菊花賞のいずれかで2勝する","The Satsuki Sho, Japan Derby and Kikka Sho\nWin at least 2 of them"
+"桜花賞、オークス、秋華賞のいずれかで2勝する","The Oka Sho, Oaks and Shuka Sho\nWin at least 2 of them"
+"大阪杯、天皇賞（春）、宝塚記念を勝利する","Win the Oosaka Hai, Tenno Sho (Spring), and Takarazuka Kinen"
+"天皇賞（秋）、ジャパンC、有馬記念を勝利する","Win the Tenno Sho (Autumn), Japan Cup, and Arima Kinen"
+"天皇賞（春）と天皇賞（秋）を勝利する","Win the Tenno Sho (Spring) and Tenno Sho (Autumn)"
```
Comment (FabulousCupcake) [inline src/data/title-requirements.csv:29]: I'm not sure about these; title requirements field have rather strict length limits. I'd check how they look like ingame and perhaps add size markup.

File: src/data/skill-desc.csv [hunk 2]
```diff
@@ -393,5 +393,5 @@
 "レース終盤に前方が詰まると\nお告げ次第で道をわずかに切り開く","<size=22>Slightly easier to find an opening if blocked at the front in Final Leg</size>"
 "レース終盤で3番手の時に負けそうに\nなると速度がわずかに上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed slightly increases</size>"
 "レースを冷静に運ぶことができると残り\n200mで実力発揮、速度が少し上がる","<size=22>After proceeding through the race calmly, your speed increases a little during the last 200m</size>"
-"最終直線で前の方で追いすがっていると\n速度がわずかに上がる",""
-"最終コーナー以降に前めで力が尽きかけ\n詰め寄られると速度がわずかに上がる",""
+"最終直線で前の方で追いすがっていると\n速度がわずかに上がる","<size=22>Slightly increase speed in the last straight when chasing someone close to the lead</size>"
+"最終コーナー以降に前めで力が尽きかけ\n詰め寄られると速度がわずかに上がる","<size=22>Slightly increase speed after the last corner when stamina runs out being one of the lead</size>"
```
Comment (FabulousCupcake) [inline src/data/skill-desc.csv:397]: Does this fit? lol  
I can check later ingame

File: src/data/title-requirements.csv (hunk 1 above)
Comment (stardustichi) [reply src/data/title-requirements.csv:29]: most of them look fine, i put size markups for some longer requirements
https://files.catbox.moe/lukckg.PNG
https://files.catbox.moe/lrcaib.PNG

File: src/data/skill-desc.csv (hunk 2 above)
Comment (stardustichi) [reply src/data/skill-desc.csv:397]: the 3 and higher star skill variants fit just fine 
https://files.catbox.moe/32h48s.PNG
these two are lower star skill variants, i doubt they even show in game but should be fine if they do

**Comment by FabulousCupcake** [comment]
Thank you!

**Pull request merged**